app/routes/send-endringsmelding.tsx

```diff
@@ -2,119 +2,142 @@ import HovedInnhold from '~/komponenter/hovedInnhold/HovedInnhold';
 import TekstBlokk from '~/komponenter/tekstblokk/TekstBlokk';
 import StegIndikator from '~/komponenter/stegindikator/StegIndikator';
 import React, { useEffect, useState } from 'react';
-import { useSpråk, useTekster } from '~/hooks/contextHooks';
+import {
+  useEndringsmeldingMottattDato,
+  useSpråk,
+  useTekster,
+} from '~/hooks/contextHooks';
 import { Button, Textarea } from '@navikt/ds-react';
-import { useNavigate } from '@remix-run/react';
+import { Form, useActionData, useNavigate, useSubmit } from '@remix-run/react';
 import Veiledning from '~/komponenter/veiledning/Veiledning';
 import css from './send-endringsmelding.module.css';
 import { ETypografiTyper } from '~/typer/typografi';
 import { ESanityMappe, ESteg } from '~/typer/felles';
 import { hentPathForSteg } from '~/utils/hentPathForSteg';
+import { sendEndringsmelding } from '~/utils/sendEndringsmelding';
+import { ActionArgs } from '@remix-run/node';
+import { EFritekstFeil } from '~/typer/fritekstfeil';
+import {
+  MAKS_INPUT_LENGDE,
+  RESPONSE_STATUS_OK,
+  SPESIAL_TEGN_REGEX,
+} from '~/konstanter/sendEndringsmelding';
+import { getSession } from '~/sessions';
+import { i18nInnhold } from '~/utils/i18n';
+
+export async function action({ request }: ActionArgs) {
+  const formData = await request.formData();
+  const endringsmelding = formData.get('endringsmelding') as string;
+  return await sendEndringsmelding(
+    endringsmelding,
+    await getSession(request.headers.get('Cookie')),
+  ).then(response => {
+    if (!response.ok) {
+      console.log('Feil i send endringsmelding');
+      throw Error('Det skjedde en feil under POST til backend');
+    }
```

Comment:
Forslag;
La denne responsen gå ned til action-staten så kan actionstaten håndtere om responsen er ok eller feil - hvis den er feil så kan du sette en state på "feilmelding" og vise den. 

Da er det mulig for brukeren å prøve sende inn på nytt igjen.

Reply:
FIkset i commit 7b5546c og med en alert her i commit b7679c9